watson-developer-cloud/discovery-components
# chore: add unit tests for elementFromPoint

#### What do these changes do/fix?

Adds unit tests for the `elementFromPoint` utility.

#### How do you test/verify these changes?

In discovery-react-components run:
`yarn test elementFromPoint` and see that tests pass.

https://github.ibm.com/Watson-Discovery/docviz-squad-issue-tracker/issues/234
